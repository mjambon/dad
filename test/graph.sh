./ad -t dot < test.yml
