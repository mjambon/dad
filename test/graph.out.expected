digraph G {
  rankdir = TB;
  bgcolor = "transparent";
  "furniture";
  "table";
  "table" -> "furniture";
  "seat";
  "seat" -> "furniture";
  "chair";
  "chair" -> "seat";
  "stool";
  "stool" -> "seat";
  "bench";
  "couch";
  "couch" -> "furniture";
}
